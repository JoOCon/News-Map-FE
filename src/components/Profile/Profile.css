.user-wraper {
  height: -webkit-fill-available;
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center; }

.user-profile {
  display: flex;
  flex-direction: column;
  background: rgba(154, 154, 154, 0.9);
  border-radius: .5em;
  padding: 1em;
  align-items: center; }

.user-profile img {
  width: 9em;
  border-radius: .5em; }

.user-profile h1 {
  margin: .5em 0; }

.user-profile h2 {
  margin: .5em 0; }
