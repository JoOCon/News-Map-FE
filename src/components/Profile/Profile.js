import React, { Component } from 'react';

class Profile extends Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return <div>dsdsdsds</div>;
  }
}

export default Profile;
