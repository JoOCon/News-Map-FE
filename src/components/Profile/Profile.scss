.user-wraper {
  height: 100%;
  width: 100%;
}

.user-profile {
  
}