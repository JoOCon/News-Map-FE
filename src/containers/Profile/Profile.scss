.user-wraper {
  height: -webkit-fill-available;
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  background: radial-gradient(#868686, #52535a);
}

.user-profile {
  display: flex;
  flex-direction: column;
  background: rgba(170, 170, 170, .9);
  border-radius: .5em;
  padding: 1em;
  align-items: center;
}

.user-profile img {
  width: 9em;
  border-radius: .5em;
}

.user-profile h1 {
  margin: .5em 0;
}

.user-profile h2 {
  margin: .5em 0;
}
