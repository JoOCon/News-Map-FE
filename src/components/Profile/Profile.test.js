import React from 'react';
import Profile from './Profile';

describe('Profile', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Profile />);
  });

  it('should matchSnapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
