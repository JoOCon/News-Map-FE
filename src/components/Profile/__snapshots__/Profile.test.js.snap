// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should matchSnapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Profile />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "dsdsdsds",
    },
    "ref": null,
    "rendered": "dsdsdsds",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "dsdsdsds",
      },
      "ref": null,
      "rendered": "dsdsdsds",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
